sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -44,6 +47,42 @@ export { delay }
 
 export { Delivery }
 
+// @public
+export type ListQueuesResult = Queue[] & {
+    _response: HttpOperationResponse & {
+        parsedBody: Queue[];
```

Comment:
Why do we want to repeat the parsed Queue information both in the outer and inner level?

Reply:
Hmm... It looks like this is the format we are using elsewhere too https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-keys/src/core/models/index.ts#L2368

Reply:
This is based on the response type used in ARM libraries.
The parsed Queue is present as part of the _response anyway, and we'll be including it per norm.